lib/queue.js

```diff
@@ -141,6 +141,13 @@ var Queue = function Queue(name, redisPort, redisHost, redisOptions){
   // Create blocking client (used to wait for jobs)
   //
   this.bclient = createClient();
+  Object.defineProperty(this, 'jobWaitCount', { get: function() {
+    return this.bclient.commandQueue.length;
+  } });
+  Object.defineProperty(this, 'whenCurrentWaitFinished', { get: function() {
+    var currentMove = this.bclient.commandQueue.peekFront()
+    return currentMove && currentMove.command.promise || Promise.resolve();
+  } });
 
```

Comment:
I think it is better to put them on the prototype, mostly so that we can follow the same standard as in the rest of the code. It becomes easier to contribute for others.